test/e2e/e2eutil/tls.go

```diff
@@ -35,7 +35,7 @@ func PreparePeerTLSSecret(clusterName, ns, secretName string) error {
 	certPath := filepath.Join(dir, "peer.crt")
 	keyPath := filepath.Join(dir, "peer.key")
 	caPath := filepath.Join(dir, "peer-ca.crt")
-	hosts := []string{fmt.Sprintf("*.%s.%s.svc.cluster.local", clusterName, ns)}
+	hosts := []string{fmt.Sprintf("*.%s.%s.svc", clusterName, ns)}
```

Comment:
Actually this is not true...
Not all k8s will expand this based on my experience..